mlflow/gluon.py

```diff
@@ -114,6 +115,26 @@ def save_model(gluon_model, path, mlflow_model=Model(), conda_env=None):
                             ]
                         }
 
+    :param signature: (Experimental) :py:class:`ModelSignature <mlflow.models.ModelSignature>`
+                      describes model input and output :py:class:`Schema <mlflow.types.Schema>`.
+                      The model signature can be :py:func:`inferred <mlflow.models.infer_signature>`
+                      from datasets with valid model input (e.g. the training dataset) and valid
+                      model output (e.g. model predictions generated on the training dataset),
+                      for example:
+
+                      .. code-block:: python
+
+                        from mlflow.models.signature import infer_signature
+                        train = df.drop_column("target_label")
+                        signature = infer_signature(train, model.predict(train))
+    :param input_example: (Experimental) Input example provides one or several instances of valid
```

Comment:
It would be nice if there were some way to get the list of supported types to show up in this docstring, e.g. when I ran 

```
>>> import mlflow.gluon
>>> help(mlflow.gluon.save_model)
```

I couldn't actually see what valid types I could pass for `input_example`. Do you know offhand if there's a way? If not I can help look for one

Reply:
Hmm, if I do help(mlflow.gluon.save_model) it expands the ModelInputExample into Union:

..., input_example: Union[pandas.core.frame.DataFrame, numpy.ndarray, dict, list] = None, ...

Reply:
Ah got it, yep I see it now (sorry for the noise, noob mistake getting used to seeing types in the actual Python method signature vs in documentation for each individual param :P)